mods/flowers/init.lua

```diff
@@ -317,4 +318,14 @@ minetest.register_node("flowers:waterlily", {
 
 		return itemstack
 	end
-})
+}
+
+local waterlily_waving_def = table.copy(waterlily_def)
+waterlily_waving_def.waving = 3
+waterlily_waving_def.drop = "flowers:waterlily"
+waterlily_waving_def.groups.not_in_creative_inventory = 1
+waterlily_waving_def.node_box.fixed[2] = -33 / 64
```

Comment:
What is this change for? Maybe both waterlilly types should be -33/64?

Reply:
Works fine without this, i prefer this line is removed so that both waterlillies have identical nodeboxes.

Reply:
When I originally wrote this (about 2 years ago) that change was required to avoid the waving waterlily vanishing periodically when viewed from below. I've just checked and you are correct, it seems this isn't needed anymore so I've removed it.

Reply:
Great thanks.